ferrocene/doc/user-manual/src/rustfmt/cli.rst

```diff
@@ -8,23 +8,54 @@
 
    .. cli:option:: --check
 
-      Run in 'check' mode. Exits with 0 if input is formatted correctly. Exits with 1 and prints a diff if formatting is required.
-
-   .. cli:option:: --emit [files|stdout|coverage|checkstyle|json]
-
-      What data to emit and how
+      Check if the input is formatted correctly.
+
+      Exits with status code 0 if input is formatted correctly.
+      Exits with status code 1 and prints a diff if formatting is not correct.
+      .. code-block::
```

Comment:
I still think this example is... too much? It doesn't add much information compared to what is explained in the text before, and it uses a lot of screen space.

Reply:
ed0f014feb1d2cd4bce738a1ecd14a78d9a97210